addons/website_sale/static/src/js/website_sale_tour_shop_deleted_archived_variants.js

```diff
@@ -46,6 +46,7 @@ tour.register('tour_shop_deleted_archived_variants', {
     {
         content: "check add to cart not possible",
         trigger: '#add_to_cart.disabled',
+        run: function () {},
```

Comment:
@seb-odoo Mmh ok for now but this is problematic that disabled links can still receive click event and do something. To think about.